2024-03-04 23:30:29 | INFO | [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:8080
2024-03-04 23:30:29 | INFO | [33mPress CTRL+C to quit[0m
2024-03-04 23:30:41 | INFO | 127.0.0.1 - - [04/Mar/2024 23:30:41] "GET / HTTP/1.1" 200 -
2024-03-04 23:30:43 | INFO | 127.0.0.1 - - [04/Mar/2024 23:30:43] "GET / HTTP/1.1" 200 -
2024-03-04 23:30:49 | ERROR | Exception on /describe_region [GET]
Traceback (most recent call last):
  File "/home/guircos/.pyenv/versions/3.11.7/lib/python3.11/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/guircos/.pyenv/versions/3.11.7/lib/python3.11/site-packages/flask/app.py", line 873, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/guircos/.pyenv/versions/3.11.7/lib/python3.11/site-packages/flask/app.py", line 892, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/guircos/.pyenv/versions/3.11.7/lib/python3.11/site-packages/flask/app.py", line 1183, in make_response
    rv = self.json.response(rv)
         ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/guircos/.pyenv/versions/3.11.7/lib/python3.11/site-packages/flask/json/provider.py", line 214, in response
    f"{self.dumps(obj, **dump_args)}\n", mimetype=self.mimetype
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/guircos/.pyenv/versions/3.11.7/lib/python3.11/site-packages/flask/json/provider.py", line 179, in dumps
    return json.dumps(obj, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/guircos/.pyenv/versions/3.11.7/lib/python3.11/json/__init__.py", line 238, in dumps
    **kw).encode(obj)
          ^^^^^^^^^^^
  File "/home/guircos/.pyenv/versions/3.11.7/lib/python3.11/json/encoder.py", line 200, in encode
    chunks = self.iterencode(o, _one_shot=True)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/guircos/.pyenv/versions/3.11.7/lib/python3.11/json/encoder.py", line 258, in iterencode
    return _iterencode(o, 0)
           ^^^^^^^^^^^^^^^^^
  File "/home/guircos/.pyenv/versions/3.11.7/lib/python3.11/site-packages/flask/json/provider.py", line 121, in _default
    raise TypeError(f"Object of type {type(o).__name__} is not JSON serializable")
TypeError: Object of type int64 is not JSON serializable
2024-03-04 23:30:49 | INFO | 127.0.0.1 - - [04/Mar/2024 23:30:49] "[35m[1mGET /describe_region HTTP/1.1[0m" 500 -
